commit c71d22f4bf76a2cd1fe430ff3d83f76847a82e53
Author: Nathan Chancellor <natechancellor@gmail.com>
Date:   Sat Mar 3 23:34:50 2018 -0700

    staging: vchiq_arm: Remove unused variable
    
    This warning appears with GCC 6.4.0 from toolchains.bootlin.com:
    
    ../drivers/staging/vc04_services/interface/vchiq_arm/vchiq_arm.c: In function ‘vchiq_open’:
    ../drivers/staging/vc04_services/interface/vchiq_arm/vchiq_arm.c:1735:7: warning: unused variable ‘ret’ [-Wunused-variable]
       int ret;
           ^~~
    
    This variable's usage was removed by commit 3c980263c592 ("staging:
    vchiq_arm: Make debugfs failure non-fatal"), making it useless.
    
    Signed-off-by: Nathan Chancellor <natechancellor@gmail.com>

diff --git a/drivers/staging/vc04_services/interface/vchiq_arm/vchiq_arm.c b/drivers/staging/vc04_services/interface/vchiq_arm/vchiq_arm.c
index e48b1ef39031..82fbefc44569 100644
--- a/drivers/staging/vc04_services/interface/vchiq_arm/vchiq_arm.c
+++ b/drivers/staging/vc04_services/interface/vchiq_arm/vchiq_arm.c
@@ -1668,7 +1668,6 @@ vchiq_open(struct inode *inode, struct file *file)
 	vchiq_log_info(vchiq_arm_log_level, "vchiq_open");
 	switch (dev) {
 	case VCHIQ_MINOR: {
-		int ret;
 		VCHIQ_STATE_T *state = vchiq_get_state();
 		VCHIQ_INSTANCE_T instance;
 
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1138_c71d22f4bf76a2cd1fe430ff3d83f76847a82e53.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1138_c71d22f4bf76a2cd1fe430ff3d83f76847a82e53.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
